app/views/assignment_swap/index.html.erb

```diff
@@ -1,20 +1,19 @@
-<% javascript_include_tag("assignment_swap/validateTwoAssignmentsSwapping") %>
```

Comment:
This wasn't doing actually doing anything. It still loaded on every page, and every solution I found to only load the JS on just this page didn't work.